standalone/src/data_conversion_layer/monitoring_frame_deserialization.cpp

```diff
@@ -53,6 +53,10 @@ FixedFields::FixedFields(DeviceStatus device_status,
 
 static constexpr double toMeter(const uint16_t& value)
 {
+  if ((value == NO_SIGNAL_ARRIVED) || (value == SIGNAL_TOO_LATE))
+  {
+    return INFINITY;
```

Comment:
How does this relate to https://en.cppreference.com/w/cpp/types/numeric_limits/infinity

Reply:
I changed it accordingly